firmware/config/engines/ford_festiva.cpp

```diff
@@ -207,10 +207,8 @@ void setFordEscortGt(DECLARE_CONFIG_PARAMETER_SIGNATURE) {
 	boardConfiguration->tunerStudioSerialSpeed = 19200;
 
 	commonFrankensoAnalogInputs(engineConfiguration);
-	setCommonNTCSensor(&engineConfiguration->clt);
```

Comment:
64 files changed OMG

can we please split into refactoring of engine settings and else separately?

Reply:
All of those changes were replace

`setThermistorConfiguration\(&engineConfiguration->clt, ([0-9\.\-]+), ([0-9\.\-]+), ([0-9\.\-]+), ([0-9\.\-]+), ([0-9\.\-]+), ([0-9\.\-]+));`
with
`engineConfiguration->clt = { $1, $4, $2, $5, $3, $6, };`

But yes, I can try to move those in to their own PR.

Reply:
let me know - I can do this or your could do this, but it should be a separate merge in my opinion

Reply:
I'll cherry pick that change to its own branch